generatejson.py

```diff
@@ -94,8 +97,10 @@ def getpkginfo(filename):
         dom = minidom.parse(pkgInfoPath)
         pkgRefs = dom.getElementsByTagName('pkg-info')
         for ref in pkgRefs:
-            pkgId = ref.attributes['identifier'].value.encode('UTF-8')
-            pkgVersion = ref.attributes['version'].value.encode('UTF-8')
```

Comment:
Caused the identifier and version keys in the bootstrap.json file to not be retrieved because JSON could not decode the byte object.